tests/sqllogictests/suites/query/window_function/expr_in_window.test

```diff
@@ -307,3 +307,51 @@ USE default
 
 statement ok
 DROP DATABASE expr_in_window
+
+statement ok
+create or replace table t4(a int, b int, c int);
+
+statement ok
+insert into t4 values (1, 1, 1),(2, 2, 1), (3, 4, 1), (5, 6, 1),(3, 2, 1),(4, 6, 1),(7,8,1),(9,10,1),(8,2,1),(3,5,1),(4,5,1),(2,3,1),(4,2,1),(2,6,1);
+
+query TT
+SELECT a, lag(a, 2) OVER (ORDER BY c) from t4;
+----
+1 NULL
+2 NULL
+3 1
+5 2
+3 3
+4 5
+7 3
+9 4
+8 7
+3 9
+4 8
+2 3
+4 4
+2 2
+
+statement ok
+set max_block_size=1;
+
+query TT
+SELECT a, lag(a, 2) OVER (ORDER BY c) from t4;
+----
+1 NULL
+2 NULL
+3 1
+5 2
+3 3
+4 5
+7 3
+9 4
+8 7
+3 9
+4 8
+2 3
+4 4
+2 2
+
```

Comment:
better set `max_block_size` to default value, prevent interference with other tests.